maint/script/build.sh

```diff
@@ -48,14 +44,12 @@ fi
 
 if [ -f "/.dockerenv" ]; then
     # Make install
-    pushd $THIS_SCRIPT_DIR/../../build/ > /dev/null
-    make install
```

Comment:
The cli command nnfusion has to be installed in system. Otherwise, the default nnfusion in /usr/local/bin will be invoked.